packages/ardrive_uploader/example/lib/main.dart

```diff
@@ -127,6 +127,7 @@ class _UploadFormState extends State<UploadForm> {
         parentFolderId: parentFolderIdController.text,
         isPrivate: false,
         type: UploadType.turbo,
+        path: null, // TODO: double check
```

Comment:
It's an example, no big deal
```suggestion
        path: null,
```